src/igvfd/types/sample.py

```diff
@@ -57,3 +57,15 @@ class CellLine(Biosample):
 class Tissue(Biosample):
     item_type = 'tissue'
     schema = load_schema('igvfd:schemas/tissue.json')
+
```

Comment:
This seems weird.  I wonder if you will want to break out the types into their own files like tissue.json, cell_line.json etc.  I feel like this merge with the collection being called technical sample but the Class being Tissue is strange

Reply:
NVM If I open the whole file, then it reads normally.